cmd/otel-allocator/README.md

```diff
@@ -1,14 +1,25 @@
 # Target Allocator
 
-The TargetAllocator is an optional separately deployed component of an OpenTelemetry Collector setup, which is used to
-distribute targets of the PrometheusReceiver on all deployed Collector instances. The release version matches the
+Target Allocator is an optional component of the OpenTelemetry Collector [Custom Resource](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/) (CR). The release version matches the
 operator's most recent release as well.
 
-In essence, Prometheus Receiver configs are overridden with a http_sd_configs directive that points to the
-Allocator, these are then loadbalanced/sharded to the collectors. The Prometheus Receiver configs that are overridden
-are what will be distributed with the same name. In addition to picking up receiver configs, the TargetAllocator
-can discover targets via Prometheus CRs (currently ServiceMonitor, PodMonitor) which it presents as scrape configs
-and jobs on the `/scrape_configs` and `/jobs` endpoints respectively.
+In a nutshell, the TA is used to manage Prometheus metrics without actually needing to have Prometheus installed in your Kubernetes cluster.
```

Comment:
I don't think this is a correct assessment.  The target allocator does no scraping or management of Prometheus metrics.  It is a mechanism for decoupling the service discovery and metric collection functions of Prometheus such that they can be scaled independently.  The collector is the component that manages Prometheus metrics without needing to install Prometheus.  The TA manages the configuration of the collector's prometheus receiver.